src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -454,7 +472,18 @@ private static void fillMethodModel(MethodModel mModel, String name, ArrayList<S
                         break;
                 }
                 if (j < comment.length()) {
-                    mModel.setDescription(mModel.getDescription() + ' ' + comment.substring(j));
+                    if (inDescription) {
+                        mModel.setDescription(mModel.getDescription() + ' ' + comment.substring(j));
+                    } else if (inExample) {
```

Comment:
This change is to preserve code indentation and new lines, which `@description` doesn't do.